### PR TITLE
added 1.99.7 reply format

### PR DESCRIPTION
**Pull request by DvirDukhan**

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -272,3 +272,26 @@ func TestParameterizedQuery(t *testing.T) {
...
+	if err != nil {
+		t.Error(err)
+	}
+	assert.Equal(t, 0, res.IndicesCreated(), "Expecting 0 index created")
+
+	res, err = graph.Query("DROP INDEX ON :user(name)")
+	if err != nil {
+		t.Error(err)
+	}
+	assert.Equal(t, 0, res.IndicesCreated(), "Expecting 1 index deleted")
```
Comment (swilly22) [inline client_test.go]: `IndicesDeleted`

File: query_result.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ const (
 	RELATIONSHIPS_DELETED   string = "Relationships deleted"
 	PROPERTIES_SET          string = "Properties set"
 	RELATIONSHIPS_CREATED   string = "Relationships created"
+	INDICES_CREATED string = "Indices created"
+    INDICES_DELETED string = "Indices deleted"
```
Comment (swilly22) [inline query_result.go]: Ident

File: query_result.go [hunk 3]
```diff
@@ -351,6 +353,14 @@ func (qr *QueryResult) RelationshipsDeleted() int {
...
 }
 
+func (qr *QueryResult) IndicesCreated() int {
+	return qr.getStat(INDICES_CREATED)
+}
+
+func (qr *QueryResult) IndicesDeleted() int {
+	return qr.getStat(INDICES_DELETED)
+}
+
```
Comment (swilly22) [inline query_result.go]: Is this really all there's to it?

File: client_test.go (hunk 1 above)
Comment (DvirDukhan) [reply client_test.go]: fixed

File: query_result.go (hunk 2 above)
Comment (DvirDukhan) [reply query_result.go]: looks ok at my machine. re-tabbed

File: query_result.go (hunk 3 above)
Comment (DvirDukhan) [reply query_result.go]: yes

**Pull request merged**